docs/pages/reference/yaml-reference.md

```diff
@@ -352,7 +352,23 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 
       * `visibility`
 
-        The default visibility for the discovery information. May be `FRAMEWORK`, `CLUSTER`, or `EXTERNAL`. If unset this defaults to `CLUSTER`. See [Mesos documentation](http://mesos.apache.org/documentation/latest/app-framework-development-guide/) on service discovery for more information on these visibility values.
+        The default visibility for the discovery information. May be `FRAMEWORK`, `CLUSTER`, or `EXTERNAL`. If unset this defaults to `CLUSTER`. See [Mesos documentation]
+        (http://mesos.apache.org/documentation/latest/app-framework-development-guide/) on service discovery for more information on these visibility values.
```

Comment:
I mean, at the moment it has:

```
See [Mesos documentation]
        (http://mesos.apache.org/documentation/latest/app-framework-development-guide/)
```

rather than:

```
See [Mesos documentation](http://mesos.apache.org/documentation/latest/app-framework-development-guide/)
```